cypress/integration/loggedOutFlows/showLoginModal.spec.js

```diff
@@ -23,9 +23,9 @@ describe('Show log in modal', () => {
   it('should show login modal for article reaction clicks', () => {
     cy.findAllByText('Test article').last().click();
 
-    cy.findByRole('checkbox', { name: 'Heart' }).as('heartReaction');
-    cy.findByRole('checkbox', { name: 'Unicorn' }).as('unicornReaction');
-    cy.findByRole('checkbox', { name: 'Saved' }).as('bookmarkReaction');
+    cy.findByTestId('like').as('heartReaction');
+    cy.findByTestId('unicorn').as('unicornReaction');
+    cy.findByTestId('readinglist').as('bookmarkReaction');
```

Comment:
```suggestion
    cy.findByRole('button', { name: 'Like' }).as('heartReaction');
    cy.findByRole('button', { name: 'React with unicorn' }).as('unicornReaction');
    cy.findByRole('button', { name: 'Add to reading list' }).as('bookmarkReaction');
```

Refactor (blocking) - change this back to `findByRole`, but update to `button` and the name added as an aria-label as in above comment. Remove the data-testids from the elements. This is preferable for the Cypress tests, because instead of testing a testid, we're testing the accessible name surfaced to our users - it just helps make the tests reflect the user experience a little bit more! 🙂